src/test/java/jalau/at18/katas/bankocr/robertof/FileReaderTest.java

```diff
@@ -0,0 +1,27 @@
+package jalau.at18.katas.bankocr.robertof;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import static org.junit.Assert.assertEquals;
+
+public class FileReaderTest {
+
+    @Test
+    public void shouldHavePath() {
+        File file = new File("src/main/java/jalau/at18/katas/bankocr/robertof/sampleFile");
+        FileReader fileReader = new FileReader(file);
+        assertEquals("src/main/java/jalau/at18/katas/bankocr/robertof/sampleFile", fileReader.getFilePath());
+    }
+
+    @Test
+    public void shouldReadFile() throws IOException {
+        File file = new File("src/main/java/jalau/at18/katas/bankocr/robertof/sampleFile");
+        FileReader fileReader = new FileReader(file);
+        ArrayList<String> tableValue = fileReader.readFile(fileReader.getFilePath());
```

Comment:
Why fileReader needs to ask itself for the path?  if the FileReader already knows the path, maybe it can directly use it after File is given in the constructor instead of sending it as method parameter. Would that make sense?